infinitic-common/build.gradle.kts

```diff
@@ -53,6 +56,9 @@ tasks {
     compileTestKotlin {
         kotlinOptions.jvmTarget = JavaVersion.VERSION_1_8.toString()
     }
+    compileTestFixturesKotlin {
+        kotlinOptions.jvmTarget = JavaVersion.VERSION_1_8.toString()
+    }
```

Comment:
I think the whole `tasks {}` block can be replaced with this to avoid too much repetition:

```
tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
    kotlinOptions.jvmTarget = JavaVersion.VERSION_1_8.toString()
}
```

Reply:
Indeed